storage/innobase/xtrabackup/src/file_utils.h

```diff
@@ -0,0 +1,28 @@
+/******************************************************
+Copyright (c) 2021 Percona LLC and/or its affiliates.
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; version 2 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
+
+*******************************************************/
+
+#ifndef FILE_UTILS_H
+#define FILE_UTILS_H
+
+/* Return a safer suffix of FILE_NAME, or "." if it has no safer
+   suffix.  Check for fully specified file names and other atrocities.
+   Warn the user if we do not return NAME. */
+/* Based on gnu tar */
```

Comment:
Please follow MySQL standard.
for example
@param[in,out]  log               redo log
@param[in,out]  contiguous_lsn    log sequence number
                                  until which all redo log has been
                                  scanned
@param[in,out]  to_lsn            LSN to stop recovery at */